pkg/api/pullrequestpayloadqualification/v1/types.go

```diff
@@ -0,0 +1,93 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PullRequestPayloadQualificationRun represents the intent to run a battery of OCP release
+// payload validating jobs
+type PullRequestPayloadQualificationRun struct {
+	metav1.TypeMeta   `json:",inline"`
```

Comment:
Given a PR, how do I find the instance of this?  How many instances can there be?

Reply:
N instances: corresponding to multiple `/payload informing 4.7` comments over time, and they could run over different tested state, and they can (but not must) run different sets of jobs

 Given a PR, you would need to search for resources with matching `PullRequestUnderTest`, I guess we could label the resource for easier search?